codebundles/istio-sidecar-check/runbook.robot

```diff
@@ -0,0 +1,129 @@
+*** Settings ***
+Documentation       Checks for Istio sidecar injection across deployments in all namespaces.
```

Comment:
We will want the documentation updated to reflect more than just sidecar injection